src/util/platform.windows.cc

```diff
@@ -54,8 +55,17 @@ std::optional<filesystem::path> platform::dataDirectory() {
 bool platform::isTTY() { return true; }
 
 std::vector<filesystem::path> platform::glob(const filesystem::path& pattern, size_t max) {
-    logger()->warn("platform::glob is not implemented on Windows");
-    return {};
+    // glob::glob returns std::filesystem::path, but we're using ghc::filesystem for compatibility
+    // reasons. this means we need to copy the paths from one vector type to another here.

```

Comment:
Their README says: `If you can't use C++17, you can integrate [gulrak/filesystem](https://github.com/gulrak/filesystem) with minimal effort.`. I have this vague memory, that I've actually done that in the past with this library, but I don't find it right now after a quick look. Something to potentially clean up later.

Reply:
Looks like it could be as simple as this: https://github.com/p-ranav/glob/issues/2